ngi_pipeline/conductor/launchers.py

```diff
@@ -25,15 +25,15 @@ def launch_analysis(projects_to_analyze, restart_failed_jobs=False,
     :param dict config: The parsed NGI configuration file; optional/has default.
     :param str config_file_path: The path to the NGI configuration file; optional/has default.
     """
-    for project in projects_to_analyze: # Get information from Charon regarding which best practice analyses to run
-        try:
-            engine = get_engine_for_bp(project, config, config_file_path)
-        except (RuntimeError, CharonError) as e:
-            LOG.error('Project {} could not be processed: {}'.format(project, e))
-            continue
-        engine.local_process_tracking.update_charon_with_local_jobs_status(config=config)
     charon_session = CharonSession()
     for project in projects_to_analyze:
+        analysis=NGIAnalysis(project, restart_failed_jobs,
+                    restart_finished_jobs, restart_running_jobs,
+                    keep_existing_data, no_qc, exec_mode,
+                    quiet, manual, config, config_file_path,
+                    generate_bqsr_bam, LOG)
```

Comment:
for clarity/robustness, it may be worthwhile to use the names for the arguments or pass them as a `**{}`